resource/labeling-app/components/text-sample/textArea.js

```diff
@@ -0,0 +1,223 @@
+import {config, UNDEFINED_COLOR, UNDEFINED_CAPTION} from "../utils/utils.js";
+
+
+const TextArea = {
+    name: 'TextArea',
+    props: {
+        text: String,
+        entities: {
+            type: Array,
+            default: () => {
+                return [];
+            }
+        },
+        selectedLabel: String,
+    },
+    computed: {
+        splittedText: function () {
+            return this.splitText(this.text);
+        }
+    },
+    methods: {
+        splitText(txt, sep=' ') {
+            return txt.split(sep).map((x) => this.sanitizeWord(x, sep)).reduce((x, y) => x.concat(y));
+        },
+        sanitizeWord(word, sep=' ') {
+            const sanitizedWordList = [];
+            if (word.match(/^[a-zA-Z0-9]+$/i)) {
```

Comment:
I actually think this regexp can be too strict for defaults since it doesn't work for non-latin alphabet. I think we could default to anything that is not punctuation

Reply:
Changed in branch "feature/add-prelabeling"